cpp/ql/test/library-tests/dataflow/dataflow-tests/test_ir.expected

```diff
@@ -17,8 +17,6 @@
 | test.cpp:110:10:110:12 | Load: (reference dereference) | test.cpp:109:9:109:14 | Call: call to source |
 | test.cpp:126:8:126:19 | Convert: (const int *)... | test.cpp:120:9:120:20 | InitializeParameter: sourceArray1 |
 | test.cpp:126:8:126:19 | Load: sourceArray1 | test.cpp:120:9:120:20 | InitializeParameter: sourceArray1 |
-| test.cpp:132:22:132:23 | Load: m1 | test.cpp:122:18:122:30 | InitializeParameter: sourceStruct1 |
```

Comment:
Were these false positives that were fixed, or false negatives that were introduced?

Reply:
I consider them to be false positives. They were both flows from `x` to `x.a`. That should propagate taint but not data flow.